internal/scheduler/scheduling_algo.go

```diff
@@ -162,7 +162,7 @@ func (l *LegacySchedulingAlgo) scheduleOnExecutor(
 ) ([]*SchedulerJob, error) {
 	ctx, cancel := context.WithTimeout(ctx, 10*time.Second)
 	defer cancel()
-	nodeDb, err := l.constructNodeDb(executor.Nodes, l.priorityClassPriorities)
+	nodeDb, err := l.constructNodeDb(executor.Nodes, l.config.Preemption.PriorityClasses)
```

Comment:
there are lots of changes to `lease.go` but hardly any changes here?  Is that because this doesn't need changes or is it because they still need to be made?

Reply:
I haven't made `scheduling_algo.go` work with preemption-to-fair-share yet. I think it may be best to do that in a follow-up PR. The existing scheduling (without preemption-to-fair-share) should "just work" though.